account_invoice_refund_reason/wizard/account_move_reversal.py

```diff
@@ -0,0 +1,24 @@
+# Copyright (C) 2019 Open Source Integrators
+# Copyright (C) 2019 Serpent Consulting Services Pvt. Ltd.
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+from odoo import api, fields, models
+
+
+class AccountMoveReversal(models.TransientModel):
+    _inherit = "account.move.reversal"
+
+    reason_id = fields.Many2one(
+        "account.invoice.refund.reason", string="Reason to credit"
```

Comment:
```suggestion
        "account.invoice.refund.reason", string="Refund Reason"
```
Perhaps using the same name in the wizard as the one in the view just to not confuse the user.